src/components/UploadPhoto/UploadPhoto.css

```diff
@@ -0,0 +1,41 @@
+.uploadPhotoContainer {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    height: inherit;  
+    justify-content: space-around;
+    width: inherit;
+}
```

Comment:
Remove .uploadPhotoContainer css because it is duplicate of the css on the AddPhoto page

Reply:
ok